sopel/run_script.py

```diff
@@ -11,6 +11,8 @@
 from __future__ import unicode_literals, absolute_import, print_function, division
 
 import sys
+
+from sopel.cli import utils
```

Comment:
This is more a general thing, but I think it would be useful to update imports as we touch files to be consistent about how `from sopel import _` statements should work.

Personally, I think that as Sopel gains more nested classes it will be clearer to do e.g. `from sopel import cli` and then refer to `cli.utils._()` so it's very obvious where something is coming from without having to refer back to the imports. The only downside is that code lines will end up being slightly longer as they need to reference more subclasses, but I would much rather have clear code that is long instead of short code that requires more work to read.

Thoughts? (We can move this discussion to a separate issue and/or IRC if you want.)

Reply:
Yes, but I'll keep it like that until I move `sopel.run_script` in `sopel.cli.run`, if you don't mind?

Because i `sopel.run_script` I have to either import `sopel.cli` or `sopel.cli.utils`, while in `sopel.cli.run` I can import `from . import utils` for example, and that'll work fine.

What I want to achieve is this:

```
# in sopel/module.py
from sopel import config

# access to config's public interface
```

and:

```
# in sopel/cli/run.py
from sopel import config, loader
from . import utils

# access to sopel.config's public interface
# access to sopel.cli.utils's cli-related public interface
```

If you see what I mean (I'm not sure to be very clear with my explanations).

Reply:
Totally cool with me. :ok_hand: